internal/backend/service.go

```diff
@@ -186,3 +186,15 @@ func (s *Service) OperateService(serviceName string, data service.OperationReque
 	response := client.streamWithRetry(path.Join(serviceEntity, serviceName)+"/operate/", "PUT", data)
 	response.Process(true)
 }
+
+func (s *Service) ScalingServiceConsent(serviceName string, data interface{}) (service.ScalingConsentResponse, error) {
+	client := newApiClient()
+
+	response := client.actionWithRetry(path.Join(serviceEntity, serviceName)+"/reactivescaledscalercomponents", "POST", data)
```

Comment:
Lets name api to `reactive-scaled-scaler-components`. This is more readable than current name

Reply:
Done